src/rpcwallet.cpp

```diff
@@ -2555,6 +2555,11 @@ Value resetmintzerocoin(const Array& params, bool fHelp)
     // update the meta data of mints that were marked for updating
     Array arrUpdated;
     for (CZerocoinMint mint : vMintsToUpdate) {
+
+        if(pwalletMain->IsCrypted() && !mint.Encrypt()){
+            LogPrintf("Error: encryption of mint failed");
```

Comment:
It would be beneficial to add context to these logs so that if there is a failure to mint we know where exactly in the code it is happening. Also you need to terminate the string with `\n`.